Sources/Identity.swift

```diff
@@ -48,6 +49,14 @@ import Foundation
 
     // MARK: Event Listeners
 
+    /// Handles events to set the advertising identifier. Called by listener registered with event hub.
+    /// - Parameter event: event containing `advertisingIdentifier` data
+    private func handleRequestContent(event: Event) {
+        state.updateAdvertisingIdentifier(event: event,
```

Comment:
What do you think about making the isAdIdEvent() a computed `Bool` var on the Event extension? Since the usage for this value is strongly tied to a given instance of Event.

Also should we add a log statement to the `else` case of the event.isAdIdEvent check? Something like:
```swift
if event.isAdIdEvent {
   state.updateAdvertisingIdentifier(...)
} else {
   Log.debug(label: IdentityConstants.LOG_TAG, "IdentityState - Event is not an Ad ID event. Not updating advertising identifier.")
}
```

I've added the check you proposed to `handleRequestContent(event:)` and updated the Event+Identity extension to include the computed variable for checking if the event is an AdId event